ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -443,6 +443,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-efs
+- as: aws-ipi-ovn-stress
```

Comment:
Generally, we add `-fN` to indicate the test frequency. E.g, f7 means run the job every 7 days. And we also use that field to generate the cron entry.

Reply:
@liangxia Thanks. I'll update the name to 'aws-ipi-ovn-stress-f30'. Is the cron ok? If not is there any rules to follow?

Reply:
We are using this shell [script](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh) to generate the cron entry.